### DISCUSSION TITLE
Allure Report Screenshots

### DISCUSSION THREAD

**Discussion by sebtechni** [seleniumbase/SeleniumBase#985] [Q&A] [answered]
Hello,

In the readme, there's a section about Allure report (https://github.com/seleniumbase/SeleniumBase#allure-reports).

I tried it but screenshots are not attached. With my tests that are not using SeleniumBase, it works.

How can I add screenshots of failed tests in a Allure report using SeleniumBase?

Thanks!

**Comment by mdmintz** [comment, marked as answer]
Hello @sebtechni, the Allure integration is a stand-alone pytest plugin integration. By default SeleniumBase saves screenshots to pytest-html reports (after all tests complete) and to the ``latest_logs/`` folder (for failing tests). If you want screenshots to appear in Allure reports, you'll need to create a custom ``tearDown()`` method and send a request in there for the screenshots to appear in Allure reports. For instructions on creating a custom ``tearDown()`` method that doesn't override the existing SeleniumBase ``tearDown()`` method, see: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/boilerplates/base_test_case.py

**Reply by sebtechni** [reply]
Thanks a lot @mdmintz, I'll take a look into that!
